CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 1.7.0 (September 21, 2020)
+ENHANCEMENTS:
+* `resource/resource_turbot_resource`: Introduced arguments `full_data` and `full_metadata` to provide a alternate of `data` and `metadata`. Now you can fully manage data and metadata, refer to resource documentation.
```

Comment:
* `resource/resource_turbot_resource`: Introduced arguments `full_data` and `full_metadata` to provide alternatives to `data` and `metadata`. These allow you to fully manage the resource data/metadata (any additional properties not in the config are deleted). Refer to resource documentation for full details.